### ISSUE TITLE
Add shared preferences into the app.

### ISSUE THREAD

**Issue by justdvnsh** [justdvnsh/AnimeClassroom#13]
We need to add shared preferences into the app, so that we can retain the instances like the UI theme of the app.

NOTE: - The shared preferences should be made like a container which can be injected anywhere we want. 

The shared preference class should be made in a module which can be injected by dagger anywhere in the app. I have added a reference . Also, the file should be in the __DI package of the app module__

```
@Module
@InstallIn(ActivityComponent::class.java)
class SharedPreference {
    // rest of the shared prefs implementation goes here.
}
```

**Comment by yash-k9** [comment]
Hi @justdvnsh, I'd like to work on this. I have a question, are we going to store only UI Theme Data?

**Comment by justdvnsh** [comment]
@yash-k9 Hi, yes as for now we are only going to store the ui theme data, but later we'll be storing a few more things liks recent searches and stuff. Happy coding.

**Comment by yash-k9** [comment]
Hi @justdvnsh, I've added Shared Preference and raised PR. Pls let me know if there are any other changes that need to be added in this issue.

**Comment by justdvnsh** [comment]
Solved by #71
